documentation/user_docs/docs/manual/Reshaping_etc.rst

```diff
@@ -269,6 +269,22 @@ The inputs are:
 - ``xremove`` is the range of display axes to remove. Follows the same format as
   ``xkeep``.
 
+.. warning::
+
+   It should be noted that masking through the ``xkeep`` and ``xremove``
+   arguments masks data based on the bin-centres and not through any
```

Comment:
```suggestion
   arguments will mask data based on the bin-centres and not through any
```
Issue here is that `masks` can be read as a noun, which is confusing. This makes a verb explicit